src/core/thread/mle_router.cpp

```diff
@@ -419,6 +419,18 @@ void MleRouter::ResetAdvertiseInterval(void)
     return;
 }
 
+#if OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
+otError Mle::AppendCslAccuracy(Message &aMessage)
+{
+    CslAccuracyTlv cslAccuracy;
+
+    cslAccuracy.Init();
+    cslAccuracy.SetAccuracy(static_cast<uint8_t>(otPlatTimeGetXtalAccuracy()));
+
+    return aMessage.Append(&cslAccuracy, sizeof(CslAccuracyTlv));
```

Comment:
Can use helper methods to  append TLVs with simple single value (if the TLV value is `uint8`, `uint16`, `uint32` or a data blob):
```suggestion
    return Tlv::AppendUint8Tlv(Message &aMessage, Tlv::kCslAccuracy, static_cast<uint8_t>(otPlatTimeGetXtalAccuracy()));
```